README.md

```diff
@@ -20,14 +16,6 @@ source .venv/bin/activate
 python3.8 -m pip install -r requirements.txt
 ```
 
-**To use a GitHub Dev Container in your browser:**
```

Comment:
Agree this is better snipped out; I'll do the same on the web project side of things

Reply:
👍